### ISSUE TITLE
Time to bump version??

### ISSUE THREAD

**Issue by sulaysumaria** [dylang/shortid#147]
I was looking at the source code and found this:

https://github.com/dylang/shortid/blob/318ff16938f92203538c6dc5b1a6cc2b4bd3d0fa/lib/build.js#L6

```js
// Ignore all milliseconds before a certain time to reduce the size of the date entropy without sacrificing uniqueness.
// This number should be updated every year or so to keep the generated id short.
// To regenerate `new Date() - 0` and bump the version. Always bump the version!
var REDUCE_TIME = 1459707606518;
```

**Comment by ai** [comment]
I will try to update it, but Shortid is not supported anymore. You should move to NanoID.

**Comment by sulaysumaria** [comment]
@ai, Thanks....

**Comment by ai** [comment]
Fixed d88eb2d

**Comment by ai** [comment]
Released in 2.2.15
